superduperdb/container/model.py

```diff
@@ -412,6 +416,7 @@ class Model(Component, PredictMixin):
     version: t.Optional[int] = None
     future: t.Optional[Future] = None
     device: str = "cpu"
+    preferred_devices: t.ClassVar[t.Sequence[str]] = ["cuda", "mps", "cpu"]
```

Comment:
1. It should be a tuple (so it's immutable)
2. Quibble: the double quotes `"` should be single quotes.

Reply:
1. Changed. 
2. Wait - are we not using [black](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings) for formatting? Not big on quibles - can we add that to the linting/formatting rules?

Reply:
We use black and ruff, but Black will only canonicalize quotes to `"`, and we only use single quotes, so we turn off their canonicalization for strings only.

[PEP 8](https://peps.python.org/pep-0008/) allows both, but primarily uses single quotes.

It was that way since before I came here but I am happy to avoid composed keys, which are more than twice as hard on your hands than a single keystroke (and I sometimes get [emacs pinky](http://xahlee.info/emacs/emacs/emacs_pinky.html)).